packages/generator/test/generators/builders/field-values-builder.test.ts

```diff
@@ -0,0 +1,224 @@
+import {FieldValuesBuilder} from "../../../src/generators/template-builders/field-values-builder"
+import {describe, it, expect} from "vitest"
+
+describe("Form Generator", () => {
+  process.env.BLITZ_APP_DIR = process.cwd()
+  const generator = new FieldValuesBuilder()
+
+  it("Should work with simple types", async () => {
```

Comment:
can you also add a test for two word field names like `orgName` to makes sure they are formatted correctly